src/opendr/perception/object_tracking_2d/siamrpn/siamrpn_learner.py

```diff
@@ -0,0 +1,569 @@
+# Copyright 2020-2022 OpenDR European Project
```

Comment:
As a general suggestion for the learner, the method order could be changed to somewhat match the order of the methods in the docs (ctor, fit, eval, infer, save, load, download) and/or maybe put the private/static and not implemented methods at the bottom, or right after to the method that uses them.

Reply:
resolved in [fa6eeb7](https://github.com/opendr-eu/opendr/pull/367/commits/fa6eeb7ee3ff2dd586bf8f351f3e666f430cc9e3)